docs/csharp/fundamentals/program-structure/top-level-statements.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: "Top-level statements - C# Programming Guide"
-description: Learn about top-level statements in C# 9 and later.
-ms.date: 03/08/2021
+title: "Top-level statements - programs without Main methods"
+description: Learn about top-level statements in C# 9 and later. You can create programs without the ceremony of a Program class and a Main method.
+ms.date: 05/14/2021
 helpviewer_keywords:
   - "C# language, top-level statements"
   - "C# language, Main method"
 ---
-# Top-level statements (C# Programming Guide)
+# Top-level statements - programs without Main methods
 
 Starting in C# 9, you don't have to explicitly include a `Main` method in a console application project. Instead, you can use the *top-level statements* feature to minimize the code you have to write. In this case, the compiler generates a class and `Main` method entry point for the application.
```

Comment:
Thanks @tdykstra 

I'll reorder for now. When I do the more detailed edits in each section, I'll rework this article so it would be logically first.